frontend/e2e/support/backend_server.py

```
@@ -0,0 +1,54 @@
+    return "192.0.2.100"
+
+
+async def _ipinfo(*_args: Any, **_kwargs: Any) -> dict[str, str]:
+    return {"asn": "AS64500", "ip": "192.0.2.100", "timezone": "UTC"}
+
+
+async def _asn(_ip: str, **_kwargs: Any) -> tuple[str, str]:
+    return "AS64500", "UTC"
+
+
+async def _mam_seen(*_args: Any, **_kwargs: Any) -> dict[str, str]:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `*_args`

(ANN401)